rep-2000.rst

```diff
@@ -547,6 +547,128 @@ Build System Support:
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.7
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+------------------------------------+
+|             |                        Required Support         |        Recommended Support         |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Package     |  Ubuntu Focal  |     MacOS**   |   Windows 10** | Debian Buster  | OpenEmbedded Thud |
++=============+================+===============+================+================+===================+
+| CMake       |     3.13.4     |     3.14.4    |     3.14.4     |      3.13.4    |       3.12.2      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| EmPY        |                                      3.3.2                                           |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Gazebo      |     11.0.0     |     11.0.0    |      N/A       |      11.0.0    |        N/A        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Ignition    |             Citadel            |      N/A       |     Citadel    |        N/A        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Ogre        |                                      1.10*                       |        N/A        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenCV      |     4.1.0      |     4.1.0     |     3.4.6*     |      3.2.0     |        3.2.0      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenSSL     |     1.1.1c     |     1.0.2r    |     1.0.2r     |      1.1.1c    |1.1.1b / 1.0.2r****|
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.0      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Python      |     3.7.5      |     3.7.3     |     3.7.3      |      3.7.3     |        3.7.2      |
```

Comment:
Focal is expected to use Python 3.8: see https://lists.ubuntu.com/archives/kubuntu-devel/2019-October/011812.html

Reply:
a9dc606